openslides_backend/action/actions/user/send_invitation_email.py

```diff
@@ -87,11 +87,18 @@ def perform(
                         result["message"] = e.message
                     except SMTPDataError as e:
                         result["message"] = f"SMTPDataError: {str(e)}"
+                    except Exception as e:
+                        result["message"] = f"Exception: {str(e)}"
 
                     if result["sent"]:
                         instance.pop("meeting_id", None)
                         write_request = self.create_write_requests(instance)
                         self.write_requests.extend(write_request)
+                    else:
+                        result["message"] = (
+                            str(result["message"])
+                            + f" Count {len(list(action_data))}, Index {self.index}"
```

Comment:
Proposal: f"Mail {self.index+1} from {len(action_data)}" for the index-part of the message.

Reply:
Done.